.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: node_js
+
+cache:
+  directories:
+    - '$HOME/.npm'
+
+notifications:
+  email: false
+
+node_js:
+  - '10'
+  - '8'
```

Comment:
Do we need Node.js 8 when `.nvmrc` states v10.16.0?

Reply:
Ad Engine is built with 8 so I think we do.